.github/workflows/fips.yml

```diff
@@ -0,0 +1,130 @@
+name: CI FIPS
+
+on:
+  push:
+    branches: [master]
+    paths-ignore:
+      - 'docs/**'
+      - '**/*.md'
+  pull_request:
+    branches: [master]
+    paths-ignore:
+      - 'docs/**'
+      - '**/*.md'
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - ubuntu-20.04
+        os_name:
+          - linux_openresty
+        test_dir:
+          - t/plugin/jwt-auth2.t t/plugin/hmac-auth2.t t/plugin/redirect.t t/plugin/http-logger.t t/plugin/clickhouse-logger.t t/plugin/openid-connect.t t/plugin/jwt-auth.t t/plugin/google-cloud-logging.t t/plugin/hmac-auth.t t/plugin/prometheus.t t/plugin/authz-keycloak.t t/plugin/batch-requests.t t/plugin/elasticsearch-logger.t
+          - t/admin/ssl* t/admin/schema.t t/admin/upstream.t t/config-center-yaml/ssl.t t/core/etcd-mtls.t t/core/config_etcd.t t/deployment/conf_server.t t/misc/patch.t
+          - t/node/grpc-proxy-unary.t t/node/upstream-keepalive-pool.t t/node/upstream-websocket.t t/node/client-mtls.t t/node/upstream-mtls.t t/pubsub/kafka.t t/router/radixtree-sni2.t t/router/multi-ssl-certs.t t/router/radixtree-sni.t t/stream-node/mtls.t t/stream-node/tls.t t/stream-node/upstream-tls.t t/stream-node/sni.t
```

Comment:
I wonder why we only run a subset of test, and is there a standard to determine what tests need to be run?

Reply:
Because only these tests involves ssl, and according to @membphis, we only need to cover a subset of test files to minimize the test duration.

Reply:
@membphis 
Could you confirm this?
What if we need to move the test files, or add a new test that also involves SSL? Do we need to sync this list too?

Reply:
I think we can reduce the test cases and reduce the difficulty of maintenance.
In addition, some notes need to be added.

Reply:
@moonming In fact, all tests related to SSL usage are included in this list.  So the question is, whether we need to slim this list?

Reply:
I think you have replied in your comments, all right?

https://github.com/apache/apisix/pull/8298/files#diff-5d1da2612215b3509570907291b3f68aa763efd84b326277239ca6312bd73155R28

Reply:
> I think you have replied in your comments, all right?
> 
> https://github.com/apache/apisix/pull/8298/files#diff-5d1da2612215b3509570907291b3f68aa763efd84b326277239ca6312bd73155R28

Yes, I did. Please read the comment in fips.yml.